ReviewMonitor/Offline/Offline.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Offline.swift
+//  ReviewMonitor
+//
+//  Created by Eliott Hauteclair on 4/01/18.
+//  Copyright © 2018 Tayal, Rishabh. All rights reserved.
+//
+
+import UIKit
+
+class Offline: UIView {
+
+    class func instanceFromNib() -> UIView {
```

Comment:
Should we have an `extension` in `AppHelper` class instead of havi g this function here?

Reply:
We could